.github/workflows/release.yml

```diff
@@ -26,10 +26,28 @@ jobs:
         with:
           version: v3.7.1
 
+      - name: Install sigstore Helm plugin
+        run: |
+          helm plugin install https://github.com/sigstore/helm-sigstore
+      - name: Install GPG Keys
+        run: |
+          cat <(echo -e "${{ secrets.GPG_PRIVATE_KEY }}") | gpg --import --batch
+          gpg --export > /home/runner/.gnupg/pubring.gpg
+          gpg --export-secret-keys > /home/runner/.gnupg/secring.gpg
```

Comment:
Not sure what these two lines are for? the import should do enough as long as you're in the gpg user. Otherwise I think we could use --homedir as a parameter to --import --batch and do away with these two lines.

Reply:
[Helm documentation](https://helm.sh/docs/topics/provenance/) calls out these steps

Reply:
Ahh, so it's to force a v1 format. Gotcha!